libexec/tgenv-upgrade

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+set -e
+
+[ -n "${TGENV_DEBUG}" ] && set -x
+source "${TGENV_ROOT}/libexec/helpers"
+
+echo "Upgrading tgenv to the latest version..."
+
+if [ -d "$HOME/.tgenv" ]; then
+  cd "$HOME/.tgenv"
+  git checkout main
+  git pull origin main
+else
```

Comment:
It would be good to do a `cd -` to return the user to their previous directory before they ran the command.

Reply:
done ✅